src/sortable.js

```diff
@@ -154,8 +164,14 @@ angular.module('ui.sortable', [])
                 ui.item.sortable.dropindex = ui.item.index();
                 var droptarget = ui.item.parent();
                 ui.item.sortable.droptarget = droptarget;
-                var attr = droptarget.attr('ng-model') || droptarget.attr('data-ng-model');
-                ui.item.sortable.droptargetModel = droptarget.scope().$eval(attr);
+
+                var droptargetScope = null;
+                angular.forEach(ui.item.sortable._connectedSortables, function(x) {
+                  if (x.element[0] === droptarget[0]) {
+                    droptargetScope = x.scope;
```

Comment:
Any reason to not return after finding the `droptargetScope`? It seems like it'd be useful, performance-wise, to bail early, but I may be missing something.